### PR TITLE
Initial draft of notes for 1st 2 tidy evaluation  vignettes

### PR DESCRIPTION
**Pull request by arthur-shaw**
Posting notes as they were during last week's presentation.

### CODE REVIEW

File: 02_tidy-evaluation-metaprogramming.Rmd [hunk 1]
```diff
@@ -1,13 +1,261 @@
...
+
+- Data frame columns are not objects in the global environment...
+- ... but we want to address them as such
+
+```{r}
+# proper search path: from environment object to column
+mean(mtcars$cyl + mtcars$am)
+
+# improper search: this is not the object you seek
+mean(mtcars$cyl + mtcars$am)
```
Comment (tanho63) [inline 02_tidy-evaluation-metaprogramming.Rmd]: was this meant to be `mean(cyl + am)` ?

**Review by tanho63 — APPROVED** [review]
reading through your notes for my own benefit - nice writeup! left a question, and I'll watch back the recording tonight probably before next mtg

File: 02_tidy-evaluation-metaprogramming.Rmd (hunk 1 above)
Comment (arthur-shaw) [reply 02_tidy-evaluation-metaprogramming.Rmd]: You're absolutely right. This was a mistake I noticed during the presentation, and that I failed to correct subsequently.

Comment (tanho63) [reply 02_tidy-evaluation-metaprogramming.Rmd]:
```suggestion
mean(cyl + am)
```

**Comment by arthur-shaw** [comment]
I belatedly commited some changes to reflect the final content of the slides/notes.

**Pull request merged**